src/sagemaker_pytorch_serving_container/torchserve.py

```diff
@@ -107,30 +103,6 @@ def start_torchserve(handler_service=DEFAULT_HANDLER_SERVICE):
     ts_process.wait()
 
 
-def _adapt_to_ts_format(handler_service):
-    if not os.path.exists(DEFAULT_TS_MODEL_DIRECTORY):
-        os.makedirs(DEFAULT_TS_MODEL_DIRECTORY)
```

Comment:
Is this not needed even though we set `--model-store MODEL_STORE` in the torchserve command?

Reply:
addressed